### ISSUE TITLE
Screen reader does not announce expanding and collapsing action for Open button in PropertyGrid ComboBox

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#1987]
**Repro steps:**
1. Create WinForms app with PropertyGrid control and target it to any object where one of the properties is the list that can be represented as ComboBox.
2. Run the app and Switch ON screen reader.
3. Try to expand and collapse the "Open" button and check whether the screen reader announces the Expanding or Collapsing action.

**Actual Result:**
Narrator/ NVDA is not reading Expanding or Collapsing action for the "Open" button.

**Expected Result:**
Narrator should read the Expanding or Collapsing action for the "Open" button while user tries to expand or collapse the list.

**DevDiv issue:**
https://dev.azure.com/devdiv/DevDiv/_workitems/edit/792617

**Comment by Zheng-Li01** [comment]
Verified the issue with .NET Core 3.1 Preview 3 -014638 build, the issue has been fixed, narrator can announce expanding and collapsing action for Open button in PropertyGrid ComboBox.

![IssueFixed](https://user-images.githubusercontent.com/38325459/68182951-f1b81200-ffd5-11e9-80dd-e3f2d8cbe51b.PNG)

**Comment by M-Lipin** [comment]
Need to add the improvements as some grid entries (for instance DisplayMember of ComboBox) not always announce the expanded/collapsed state. And also need to consider changing accessible control type of such entries to ComboBox.

**Comment by M-Lipin** [comment]
Reopening the issue to make corresponding changes.

**Comment by Tanya-Solyanik** [comment]
I'm not sure if this is a real accessibility issue.
1. the original iteration of the fix was signed off by the accessibility testers.
2. the current behavior is a "feature" of narrator. When narrator receives too many events, it discards some of them in order to catch up with the user actions. It does not mean that we should reduce the number of events, or force announcements that are trailing behind user actions. Notifications are a different accessibility feature, and accessibility tools can potentially handle them differently. 
3. according to Sergey's investigation, Office behaves similarly

**Comment by merriemcgaw** [comment]
I'd like ComboBoxes hosted in a PropertyGrid to announce Expand/collapse - but that I mean as the ComboBox level, and not the individual button level within that.  Unless is a compliance related bug I haven't seen yet, I don't think we have anything we have to do here. Good to know we're like Office and we can use that as a data point in the future should a related bug come up. 

I'm going to go ahead and close this issue, but feel free to reopen if someone feels super strongly about it.
